v4/source/posts.yaml

```
@@ -216,6 +216,43 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  '/user/{user_id}/posts/{post_id}/set_unread':
+    post:
+      tags:
+        - posts
+      summary: Mark as unread from the post.
+      description: |
+        Mark a channel as being unread from a given post.
+        ##### Permissions
+        Must have `read_channel` permission for the channel the post is in or if the channel is public, have the `read_public_channels` permission for the team.
+        Must have `edit_user` permission if the user is not the one marking the post for himself.
+
+        __Minimum server version__: 5.14
```

```suggestion
        __Minimum server version__: 5.16
```

This will be next release right?